config/brakeman.ignore

```diff
@@ -0,0 +1,155 @@
+{
```

Comment:
discussed in person but for posterity: we might not need this as Github adds comments inline with the PR

Reply:
Done, I didn't even realise GitHub had this level of integration 💡